src/hps_align/survey/_fixture.py

```diff
@@ -0,0 +1,324 @@
+
+import numpy as np
+
+from ._utils import *
+from ._parser import Parser
+from ._cli import app
+
+
+class Fixture:
+    """Fixture for measuring sensors
+
+    The fixture is used to measure the sensors. It consists of a base plane, a ball frame and a pin frame.
+    The pins are used to align the sensors and have the same distance as the uchannel pins.
+    Additionally, the pin frame in the fixture is defined analogously to the uchannel pin frame.
+    The pin positions cannot be used as a reference for meaurement once the sensor is installed.
+    The fixture ball frame is used as a reference frame for the sensor measurements.
+    Importantly, the fixture ball frame is not the same as the uchannel ball frame.
+    Here, the ball frame is defined by three balls on the outside of the fixture.
+
```

Comment:
Same comment as earlier, may be helpful may be not.

```suggestion
    .. image:: _static/fixture.png
       :width: 600
       :alt: Diagram Showing Ball and Pin Frame Coordinate Systems for Fixture

```